src/components/categories.js

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 
 function Categories() {
   return (
-    <div><h1>This is categories sections</h1></div>
+    <div>
+      <h1>This is categories sections</h1>
+    </div>
   );
 }
```

Comment:
- [ ] This link to your [github profile](https://github.com/wolterbwambale/bookstore) in your readme file is broken, please update it